### PR TITLE
Feature#10 frontend page model company branches

### PR DESCRIPTION
**Pull request by willcab**
# Description
implemented the view CRUD company branches
 -for each show all company names in the index
 -form to edit / new company branches
 - links to redirect to others method controller

### CODE REVIEW

**Comment by dagp-ucol** [comment]
Approved

**Comment by claunicole** [comment]
Looks good to me

**Review by claunicole — APPROVED** [review]

File: app/app/views/company/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <%= form.label :name%>
+    <%= form.text_field :name%>
+  </div>
+
+  <div>
+    <%= form.label :address%>
+    <%= form.text_area :address%>
+  </div>
+  <%= form.submit%>
+<%= end %>
```
Comment (JuanequeX) [inline app/app/views/company/_form.html.erb:17]: Add a blank line on the of this file, please!

File: app/app/views/company/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<h2>Edit company details</h2>
+
+<%= render 'form' %>
```
Comment (JuanequeX) [inline app/app/views/company/edit.html.erb:3]: Add a blank line on the of this file, please!

File: app/app/views/company/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<h1>Index Company Branches</h1>
+
+<ul>
+<% @companys.each do |company| %>
+  <li>
+  <%= link_to @company.name, company_path %>
+  </li>
+<% end %>
+</ul>
```
Comment (JuanequeX) [inline app/app/views/company/index.html.erb:9]: fix the identation of two spaces and Add a blank line on the of this file, please!

File: app/app/views/company/new.index.html.erb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<h2>New Company Branche</h2>
+
+<%= render 'form' %>
```
Comment (JuanequeX) [inline app/app/views/company/new.index.html.erb:3]: Add a blank line on the of this file, please!

File: app/app/views/company/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<h1>Company <%= @company.name %> </h1>
+<%= link_to 'edit company', edit_company_path(@company) %>
+<%= link_to t('.delete'), product_path(@product), data:{
+    turbo_method: :delete,
+    turbo_confirm: 'Are you sure?'
+} %>
```
Comment (JuanequeX) [inline app/app/views/company/show.html.erb:6]: Add a blank line on the of this file, please!

**Pull request closed without merging**